src/doc/manifest.md

```diff
@@ -720,19 +720,28 @@ other copies. The syntax is similar to the `[dependencies]` section:
 [patch.crates-io]
 foo = { git = 'https://github.com/example/foo' }
 bar = { path = 'my/local/bar' }
+
+[dependencies.baz]
+git = 'https://github.com/example/baz'
+
+[patch.'https://github.com/example/baz']
+baz = { git = 'https://github.com/example/patched-baz', branch='my-branch' }
```

Comment:
Micronit: let's add spaces around `=` in `branch='my-branch`, to be consistent with other examples here.

Reply:
Oops, of course.  Added them.